src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -62,7 +62,7 @@ public void TestThreadCount()
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.OSX)]
+        [PlatformSpecific(TestPlatforms.OSX)] // OSX throws PNSE from StartTime
```

Comment:
Hmm.  We may want to revisit this behavior; I'm not sure throwing in this case is the best option.  But that's not relevant to your PR.